src/general/OptionsMenu.tscn

```diff
@@ -607,6 +608,13 @@ __meta__ = {
 "_edit_use_anchors_": false
 }
 
+[node name="TranslationSiteButton" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Sound/VBoxContainer"]
```

Comment:
Since the issue calls for it, the button should be placed inside the language dropdown's HBoxContainer so it's positioned next to the Reset button.

Reply:
I think the button should be below the language dropdown, just in case the button text is long, don't want to squash the language button too much.

Reply:
Ah yeah that's a valid concern, this is fine then.